tests/www/views/test_views_dagrun.py

```diff
@@ -60,17 +60,12 @@ def init_blank_dagrun():
 
     This really shouldn't be needed, but tests elsewhere leave the db dirty.
     """
-    with create_session() as session:
-        session.query(DagRun).delete()
-        session.query(TaskInstance).delete()
+    clear_db_runs()
 
 
 @pytest.fixture(autouse=True)
 def reset_dagrun():
-    yield
-    with create_session() as session:
-        session.query(DagRun).delete()
-        session.query(TaskInstance).delete()
+    clear_db_runs()
```

Comment:
I think the `yield` is necessary for this fixture?

Reply:
Oops, thanks for spotting this.